src/python/pants/backend/jvm/subsystems/zinc.py

```diff
@@ -40,29 +53,35 @@ def sbt_jar(name, **kwargs):
       cls.register_jvm_tool(register,
                             'zinc',
                             classpath=[
-                              JarDependency('org.pantsbuild', 'zinc_2.10', '0.0.5'),
-                            ],
-                            main=Zinc.ZINC_COMPILE_MAIN,
-                            custom_rules=shader_rules)
+                              JarDependency('org.pantsbuild', 'zinc-compiler_2.11', '0.0.5'),
+                            ])
 
       cls.register_jvm_tool(register,
                             'compiler-bridge',
                             classpath=[
-                              sbt_jar(name='compiler-bridge_2.10',
-                                      classifier='sources',
-                                      intransitive=True)
+                              ScalaJarDependency(org='org.scala-sbt',
+                                                name='compiler-bridge',
+                                                rev=zinc_rev,
+                                                classifier='sources',
+                                                intransitive=True),
                             ])
       cls.register_jvm_tool(register,
                             'compiler-interface',
                             classpath=[
-                              sbt_jar(name='compiler-interface')
+                              JarDependency(org='org.scala-sbt', name='compiler-interface', rev=zinc_rev),
                             ],
-                            # NB: We force a noop-jarjar'ing of the interface, since it is now
-                            # broken up into multiple jars, but zinc does not yet support a sequence
-                            # of jars for the interface.
+                            # NB: We force a noop-jarjar'ing of the interface, since it is now broken
+                            # up into multiple jars, but zinc does not yet support a sequence of jars
+                            # for the interface.
                             main='no.such.main.Main',
                             custom_rules=shader_rules)
 
+      cls.register_jvm_tool(register,
+                            Zinc.ZINC_EXTRACTOR_TOOL_NAME,
```

Comment:
Seems arbitrary that this is the only tool we register here whose name is in a symbolic constant.